cli/pcluster/update/update_command.py

```diff
@@ -206,3 +209,47 @@ def _restore_desired_size(cfn_params, stack_name, scheduler):
     # If there are compute nodes running, preserve current desired capacity (if possible)
     if int(cfn_params["MinSize"]) <= desired_capacity <= int(cfn_params["MaxSize"]):
         cfn_params["DesiredSize"] = str(desired_capacity)
+
+
+def _check_cluster_models(base_config, target_config, cluster_template):
+    """
+    Check if the cluster models of the two configurations are compatible with the update operation.
+
+    If target configuration is SIT and base is HIT a conversion is required.
+    """
+    same_cluster_model = target_config.cluster_model == base_config.cluster_model
+    if not same_cluster_model:
+        conversion_supported = (
+            True
+            if (target_config.cluster_model == ClusterModel.SIT and base_config.cluster_model == ClusterModel.HIT)
```

Comment:
Minor: it can be simplified to: 
```python
conversion_supported = target_config.cluster_model == ClusterModel.SIT and base_config.cluster_model == ClusterModel.HIT
```